app/views/classified_listings/index.html.erb

```diff
@@ -32,15 +32,15 @@
     data-allcategories="<%= categories_for_display.to_json %>"
     <% if @displayed_classified_listing %>
       data-displayedlisting="<%= @displayed_listing_json %> "
-    <% end %>
-  >
+    <% end %>>
```

Comment:
It seems like one of our Husky hooks insists on reformatting this 🤷

Reply:
let the tool do they :D